### ISSUE TITLE
File "missing" while training 

### ISSUE THREAD

**Issue by omkarp1013** [ultralytics/yolov5#5576]
### Search before asking

- [X] I have searched the YOLOv5 [issues](https://github.com/ultralytics/yolov5/issues) and [discussions](https://github.com/ultralytics/yolov5/discussions) and found no similar questions.

### Question

I'm currently training YOLOv5, and while scanning my testing (val) set, I get the message: "500 found, 31 missing, 0 empty, 0 corrupt" — what does the 31 missing mean, and how can I make this right? Additionally, when I am training, I see that there are only 703 images being trained (e.g. 703/703) rather than the entire training set, which is 2811 images...is this some sort of 4-fold cross-validation? Thank you!

**Comment by Zengyf-CVer** [comment]
@pathmarkoak 
First of all, there are several points in your question that need to be improved:
1. What is the `batch size` set in your verification command? 1, 4, 16, or what?
2. Double-check whether there are unlabeled pictures in your verification set, or pictures with incorrect annotations.

**Comment by omkarp1013** [comment]
@Zengyf-CVer 
Thank you for the response. The `batch_size` set in my command is 4. Also, I have double checked that there are 0 unlabeled pictures and pictures with incorrect annotations in my verification set, so I am not sure what is wrong with the model.

**Comment by Zengyf-CVer** [comment]
@pathmarkoak 
When the batch size is 4, `2811/4=703`.

**Comment by omkarp1013** [comment]
@Zengyf-CVer 
Got it, thank you so much! Would you know why there is a certain amount of images "added" to my validation set? According to 531 total images, but only 500 labels. Referring to the attached image; my output says 531 "Images" but 500 "Labels" (this is an outdated screenshot, so "Targets" is used instead of "Labels."

**Comment by Zengyf-CVer** [comment]
@pathmarkoak 
I didn't see your screenshot. Did you send the picture successfully?

**Comment by omkarp1013** [comment]
Sorry about that; it is attached in this comment. 
![image](https://user-images.githubusercontent.com/43706200/140862623-93377bad-dcab-450e-96be-6346da495d9f.png)

**Comment by Zengyf-CVer** [comment]
@pathmarkoak 
I understand what you mean. Normally, in the verification set, if the number of images is less than or equal to the number of tags, yours is greater than. Therefore, I think there is a problem with your custom validation set. Please try to redistribute and make your own training set and validation set. Reference: https://github.com/ultralytics/yolov5/issues/5559#issuecomment-963023172

**Comment by Zengyf-CVer** [comment]
@pathmarkoak 
I have verified on 5 custom data sets, and there is no missing situation, indicating that there is a problem with your verification set, please re-divide it.

**Comment by omkarp1013** [comment]
@Zengyf-CVer 
That makes sense, thank you. But, what is interested is that I only have 500 total images in my validation set, so I am not sure why YOLO thinks there are more images.
